site/app/templates/plagiarism/Plagiarism.twig

```diff
@@ -3,7 +3,7 @@
     <div class="nav-buttons">
         <a class="btn btn-primary"
            href="{{ new_plagiarism_config_link }}"
-        >+ Configure New Gradeable for Plagiarism Detection</a>
+        ><i class="fas fa-plus"></i> Configure New Gradeable for Plagiarism Detection</a>
```

Comment:
Is it just me, or should there be a tiny bit of space/padding between the + icon and the text on this button? 
![image](https://user-images.githubusercontent.com/71195502/122274244-59238c80-ceb0-11eb-8820-f9c128a6215b.png)